packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts

```diff
@@ -79,10 +79,9 @@ export const erc20FeeProxyArtifact = new ContractArtifact<ERC20FeeProxy>(
           address: '0x612cF8a29A9c8965a5fE512b7463165861c07EAa',
           creationBlockNumber: 5216414,
         },
-        // Temporary address used to test
         fuse: {
           address: '0xee07ef5B414955188d2A9fF50bdCE784A49031Fc',
-          creationBlockNumber: 11068489,
+          creationBlockNumber: 15306858,
```

Comment:
why?
https://explorer.fuse.io/tx/0x7c3722b8dd38c3002644c137090f3cfff72e8f362eedbad004df9234345b92af/internal-transactions

Reply:
because that was the temporary test I've made to demonstrate manually Request contract to Fuse during our first meeting.

Reply:
https://github.com/RequestNetwork/requestNetwork/pull/758/files

Reply:
yes but the creation block was right, why change it?